addons/sale/data/sale_data.xml

```diff
@@ -16,5 +16,17 @@
             <field name="key">sale.default_confirmation_template</field>
             <field name="value" ref="sale.mail_template_sale_confirmation"/>
         </record>
+
+        <record model="ir.cron" id="send_invoice_cron">
+            <field name="name">automatic invoicing: send ready invoice</field>
+            <field name="model_id" ref="payment.model_payment_transaction" />
+            <field name="state">code</field>
+            <field name="code">model._cron_send_invoice()</field>
+            <field name="user_id" ref="base.user_root" />
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+        </record>
```

Comment:
The cron should be disabled unless `sale.automatic_invoice` is enabled.
Otherwise it would run for nothing or have unexpected side-effects.

Reply:
In that case I suggest to check the parameter self.env['ir.config_parameter'].sudo().get_param('sale.automatic_invoice') 
at the beginning of _cron_send_invoice

Reply:
Ideally, should be done in a post-init-hook and `set_values` override, to really disable the cron when not useful (like it's done for `ir_cron_lead_enrichment` cron).
Anyway, I believe it's more the infra team that asks for this kind of changes :sweat_smile: so the check in the cron method is enough for me :).

Reply:
even if it temporary and it will be change in the near future version let's do it right since v15.0  will live long